ladybug/hourlyplot.py

```diff
@@ -0,0 +1,424 @@
+# coding=utf-8
+from __future__ import division
+
+from .analysisperiod import AnalysisPeriod
+from .datacollection import HourlyContinuousCollection, HourlyDiscontinuousCollection
+from .graphic import GraphicContainer
+from .legend import LegendParameters
+
+from ladybug_geometry.geometry2d.pointvector import Point2D, Vector2D
+from ladybug_geometry.geometry2d.line import LineSegment2D
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry2d.polyline import Polyline2D
+from ladybug_geometry.geometry2d.mesh import Mesh2D
+from ladybug_geometry.geometry3d.pointvector import Point3D, Vector3D
+from ladybug_geometry.geometry3d.line import LineSegment3D
+from ladybug_geometry.geometry3d.plane import Plane
+from ladybug_geometry.geometry3d.polyline import Polyline3D
+from ladybug_geometry.geometry3d.mesh import Mesh3D
+
+
+class HourlyPlot(object):
+    """Module for visualization of houlry data collections.
+
+    Args:
+        data_collection: A HourlyContinuousCollection or HourlyDiscontinuousCollection
+            which will be used to generate the hourly plot.
+        legend_parameters: An optional LegendParameter object to change the display
+            of the HourlyPlot (Default: None).
+        base_point: An optional Point3D to be used as a starting point to generate
+            the geometry of the plot (Default: (0, 0, 0)).
+        x_dim: An optional number to set the X dimension of the mesh cells (Default: 1).
+        y_dim: An optional number to set the Y dimension of the mesh cells (Default: 4).
+        z_dim: An optional number to set the Z dimension of the entire chart. This
+            will be used to make the colored_mesh3d of the chart vary in the Z
+            dimension according to the data. The value input here should usually be
+            several times larger than the x_dim or y_dim in order to be noticable
+            (e.g. 100). If 0, the colored_mesh3d will simply be flat. (Default: 0).
+
+    Properties:
+        * data_collection
+        * legend_parameters
+        * base_point
+        * x_dim
+        * y_dim
+        * z_dim
+        * colored_mesh2d
+        * colored_mesh3d
+        * legend
+        * chart_border2d
+        * chart_border3d
+        * hour_lines2d
+        * hour_lines3d
+        * hour_label_points2d
+        * hour_label_points3d
+        * hour_labels
+        * month_lines2d
+        * month_lines3d
+        * month_label_points2d
+        * month_label_points3d
+        * month_labels
+        * title_text
+        * lower_title_location
+        * upper_title_location
+        * analysis_period
+        * values
+        * value_colors
```

Comment:
What about `colors`?

Reply:
The only reason why I didn't do that originally is that I didn't want people confusing it with the `legend_parameter.colors`.  But maybe this really isn't as confusing as I thought. Ok, I'll change it.